lib/active_merchant/billing/gateways/quickpay/quickpay_v4to7.rb

```diff
@@ -83,7 +83,7 @@ def store(creditcard, options = {})
         post = {}
 
         add_creditcard(post, creditcard, options)
-        add_amount(post, 0, options) if @protocol >= 7
```

Comment:
This says add_amount ... how was the _amount_ getting added if the protocol wasn't >= 7?  (Is it getting added twice now?)